feedback-module/src/components/common/Module.js

```diff
@@ -1,37 +1,89 @@
 import React, { useEffect, useState } from "react";
-import { GridContainer } from "@trussworks/react-uswds";
+import { GridContainer, Grid, Form } from "@trussworks/react-uswds";
 
-import { MODULE_CONTAINER_STYLE } from "../../assets/styling_classnames";
+import {
+  MODULE_CONTAINER_STYLE,
+  SCREEN_CONTAINER_STYLE,
+  H1_STYLE,
+  H1_WHITE_STYLE,
+  FORM_STYLE,
+} from "../../assets/styling_classnames";
+import { SCREENS } from "../../assets/constants";
 import Header from "./Header";
-import Screen1 from "../pages/Screen1";
-import Screen2 from "../pages/Screen2";
-import Screen3 from "../pages/Screen3";
-import Screen4 from "../pages/Screen4";
+import ModuleButton from "./Button";
+import CheckboxList from "../CheckboxList";
+import TextboxList from "../TextboxList";
 
 function Module() {
   const [feedback, setFeedback] = useState({});
+  const [userInfo, setUserInfo] = useState({});
+  const [screen, setScreen] = useState(SCREENS.feedback_type);
+  const [checkedFields, setCheckedFields] = useState(null);
+  const [otherField, setOtherField] = useState("");
+  const [inputQuestions, setInputQuestions] = useState();
 
   useEffect(() => {
+    if (screen.checkboxes) {
```

Comment:
you could also write this as screen.checkboxes && so that you don't need the entire if statement. Both work not 100% positive which looks cleaner